resolwe_bio/tools/get_species.py

```diff
@@ -0,0 +1,34 @@
+#!/usr/bin/env python3
+"""For a given feature, get Species using gene information knowledge base."""
+import argparse
+import resdk
+
+
+def parse_arguments():
+    """Parse command line arguments."""
+    parser = argparse.ArgumentParser(description="For a given gene/feature id and source, return Species.")
+    parser.add_argument('source', type=str, help="Source database.")
+    parser.add_argument('feature_ids', type=str, nargs="+", help="Feature IDs.")
+    return parser.parse_args()
+
+
+def main():
+    """Invoked when run directly as a program."""
+    args = parse_arguments()
+
+    res = resdk.Resolwe(url='https://qa.genialis.com')
```

Comment:
@jkokosar, refactor the script to support server address as a command line parameter. @tjanez, @dblenkus discuss how to expose the Resolwe server address in the Jinja execution engine as a template tag or template variable. I guess the address has to be configurable and will be different for each deploy.